pkg/runtime/framework/plugins/torch/torch.go

```diff
@@ -71,14 +73,20 @@ func (t *Torch) EnforceMLPolicy(info *runtime.Info, trainJob *trainer.TrainJob)
 	// TODO (andreyvelich): Add validation to check that TrainJob doesn't have "PET_" envs.
 	// TODO (andreyvelich): We should validate that envs from different plugins don't conflict with each other.
 	// Ref: https://github.com/kubeflow/trainer/pull/2308#discussion_r1823229940
+	var numProcPerNodeVal string
+	if numProcPerNode.Type == intstr.Int {
+		numProcPerNodeVal = strconv.Itoa(int(numProcPerNode.IntVal))
+	} else {
+		numProcPerNodeVal = numProcPerNode.StrVal
+	}
```

Comment:
Should just always take StrVal from numProcPerNode here since we assign it to env after ?